src/Components/Shared/test/TestWebAssemblyJSRuntimeInvoker.cs

```diff
@@ -23,6 +23,12 @@ public TestWebAssemblyJSRuntimeInvoker(string environment = "Production")
                     return (TResult)(object)_environment;
                 case "Blazor._internal.getConfig":
                     return (TResult)(object)null;
+                case "Blazor._internal.navigationManager.getBaseURI":
+                    var testUri = "https://www.example.com/awesome";
```

Comment:
TBH this is a strange base URI, since base URIs normally always end with a `/`. If a base URI doesn't end with a slash, it's functionally equivalent to the same string truncated after the last slash.

It might not make any difference for these tests but just raising this for clarity :)

Reply:
Oh wait, looks like this is intentional! I see there's a test that shows the part after the slash is truncated.

It's not a big deal but might be worth considering changing this to something like:

```suggestion
                    var testUri = "https://www.example.com/part-after-last-slash-will-be-truncated";
```

... so it's clearer to other devs in the future.